templates/provider/hmc/templates/webhooks.yaml

```diff
@@ -28,6 +28,28 @@ webhooks:
         resources:
           - managedclusters
     sideEffects: None
+  - admissionReviewVersions:
+      - v1
+      - v1beta1
+    clientConfig:
+      service:
+        name: {{ include "hmc.webhook.serviceName" . }}
+        namespace: {{ include "hmc.webhook.serviceNamespace" . }}
+        path: /mutate-hmc-mirantis-com-v1alpha1-multiclusterservice
+    failurePolicy: Fail
+    matchPolicy: Equivalent
+    name: mutation.multiclusterservice.hmc.mirantis.com
+    rules:
+      - apiGroups:
+          - hmc.mirantis.com
+        apiVersions:
+          - v1alpha1
+        operations:
+          - CREATE
+          - UPDATE
+        resources:
+          - multiclusterservices
+    sideEffects: None
```

Comment:
You're right, the mutation webhook is not needed. I guess I added it because ManagedCluster also had a mutation webhook and I couldn't find anywhere in code that ManagedCluster was being mutated. Maybe it is and I missed it? Otherwise ManagedCluster probably doesn't need the mutation webhook as well.

Reply:
Unfortunately, I do not have an answer to this question. :) But you are right; the `mc` resource does not require its mutation handler either.